README.md

```diff
@@ -1,7 +1,16 @@
 # TFExplorer
+[![Build Status](https://travis-ci.org/hardlyhuman/TFExplorer.svg?branch=master)](https://travis-ci.org/hardlyhuman/TFExplorer)
+![alt text](https://patrolavia.github.io/telegram-badge/chat.png "Logo title")
```

Comment:
You will need something like [![alt text](https://patrolavia.github.io/telegram-badge/chat.png "Logo title")](<link to telegram group) to enable users to join grp on clicking the image